test/functional/interface_ipc.py

```diff
@@ -188,10 +198,56 @@ async def async_routine():
             check_opts = self.capnp_modules['mining'].BlockCheckOptions()
             template = await mining.result.createNewBlock(opts)
             block = await self.parse_and_deserialize_block(template, ctx)
-            coinbase = await self.parse_and_deserialize_coinbase_tx(template, ctx)
-            balance = miniwallet.get_balance()
+
+            self.log.debug("Build coinbase transaction using getCoinbase()")
```

Comment:
Maybe extract this part into a separate method; that can also serve as an example on how to build the cb tx from the getCoinbase() result, and leave the flow of this test more readable